client/src/pages/Lobby.js

```diff
@@ -97,6 +97,30 @@ const Lobby = () => {
 
   if (!open) return <Redirect to='/dashboard' />;
 
+  const addUserAndQuestions = async () => {
+    if (Object.keys(userData).length === 1) {
+      await addInterviewQuestions(roomId);
+    } else {
+      if (creatorId) {
+        userData.forEach(async (user) => {
+          if (user._id !== creatorId) {
+            await addUserToInterview({ userId: user._id, roomId });
+            await addInterviewQuestions(roomId);
+          }
+        })
+      }
+    }
+  }
+
+  if (startInterview) {
+    addUserAndQuestions();
+    setInterval(function () {
+      history.push({
+        pathname: `/interview/${roomId}`
+      })
+    }, 500);
```

Comment:
Rather than guessing at a time for this, maybe we should await addUserAndQuestions and then push the pathname